src/VisualStudio/VisualStudioInteractiveComponents/VisualStudioInteractiveComponents.csproj

```diff
@@ -112,9 +112,11 @@
       <Project>{20BB6FAC-44D2-4D76-ABFE-0C1E163A1A4F}</Project>
       <Name>VisualStudioInteractiveWindow</Name>
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
-      <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup</IncludeOutputGroupsInVSIX>
-      <IncludeOutputGroupsInVSIXLocalOnly>DebugSymbolsProjectOutputGroup</IncludeOutputGroupsInVSIXLocalOnly>
-      <ForceIncludeInVSIX>true</ForceIncludeInVSIX>
+      <Private>False</Private>
```

Comment:
As we discussed offline, this is our way of conveying the VSIX dependency to msbuild.